### ISSUE TITLE
Custom node categories are wrongfully displayed as Create type nodes

### ISSUE THREAD

**Issue by andydandy74** [DynamoDS/Dynamo#5613]
![query2create](https://cloud.githubusercontent.com/assets/3014437/10754533/96499908-7c98-11e5-98d8-db38b2af4fe7.PNG)
As you can notice in the node description, the node is categorized as a Query type node. In the browser it is displayed as a Create type node, though. I have noticed this for Actions type nodes as well, e.g. ```Dimension.SetType``` in package Clockwork. As far as I can tell this *only* happens if a given Category contains a single node, but I haven't tested this thoroughly so I'm not sure if this is the only scenario where a node is graphically miscategorized.
0.9 RC1

**Comment by ThomasMahon** [comment]
Still a problem in v1.2.0, see #7244

**Comment by Racel** [comment]
@ramramps @mjkkirschner @sharadkjaiswal @aparajit-pratap  - Any thoughts on implementation? I was thinking we might want to try to address this at the same time we do the library cleanup work.

**Comment by ramramps** [comment]
@Racel Probably a bug when we classify the nodes. I will check this today.

**Comment by Racel** [comment]
Thanks @ramramps - Keep me posted. We can make a new YT and talk about adding it to sprint.

**Comment by aparajit-pratap** [comment]
@Racel I think there are related bugs in the library UI, namely:
[Bug in library view when adding both custom nodes and ZT nodes under the same category] (https://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-10323)
[When "Enable T-Splines" menu item is selected all T-Splines nodes do not appear in the library UI] (https://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-10159)

**Comment by smangarole** [comment]
@aparajit-pratap @kronz @Racel - This needs investigation after the new library changes. Should we file a task in the backlog for these specific issues?
Thanks

**Comment by aparajit-pratap** [comment]
@smangarole yes, I think it would be worthwhile to test these issues against the new library UI to make sure they are addressed.
